### ISSUE TITLE
Need Help! Could you provide the dev_gold_schema.json file for BIRD dataset?

### ISSUE THREAD

**Issue by HOU-SZ** [wbbeyourself/MAC-SQL#25]
Hi, there is a line of code mentioned the gold_schema in the run.py https://github.com/wbbeyourself/MAC-SQL/blob/dc939567015d46e615b354fdcdfd846bb07dd435/run.py#L50

Could you please the dev_gold_schema.json file since I need to run some experiments on that.

Thanks a lot!

**Comment by wbbeyourself** [comment]
Yes, in the google drive or baidu disk (link in the readme), data.zip, you can download it.  under the folder data.zip\data\bird.
